conf/application.conf

```diff
@@ -53,6 +53,8 @@ feature.ur-banner.enabled = false
 ur-banner.url = "https://www.surveygizmo.com/s3/4856606/HMRC-Letter-Study-7"
 http-verbs.retries.ssl-engine-closed-already.enabled = true
 
+contact-frontend.serviceId = "TES"
```

Comment:
where is this used?

Reply:
required for this helper.....
https://github.com/hmrc/play-frontend-hmrc#helping-users-report-technical-issues